### PR TITLE
Remove mixin border-radius from sass

### PR DESCRIPTION
**Pull request by brunowego**
Boostrap not have more border-radius mixin.

### CODE REVIEW

**Comment by fk** [comment]
Thanks @brunowego! I guess you're right ;-) – there is no `border-radius` mixin in Bootstrap itself. While I'm not too familiar with the Bootstrap 2 version of select2-bootstrap-css in `master`, I suppose it uses the Compass `border-radius` mixin.

From what I see, `master` needs some more work so that it actually compiles Sass – at least it didn't work for me when I gave it a quick spin right now. I'll make sure to include your patch when I get to fix things over there.

**Comment by fk** [comment]
OK, so here's what's going on in the `master` branch for Bootstrap 2 in regards to your pull request:

https://github.com/jlong/sass-twitter-bootstrap ([v2.3.2](https://github.com/jlong/sass-bootstrap-defunct/releases/tag/v2.3.2)) is installed via Bower so that `lib/build.scss` can pull in the Sass versions of Bootstraps variables and mixins; the mixins _do_ include a [`box-shadow` mixin](https://github.com/jlong/sass-bootstrap-defunct/blob/c4d7cbce669f4f6e5838e75e847a443c9f59bb37/lib/_mixins.scss#L247). Please forget what I said about Compass in my previous comment.

select2-bootstrap-css also provides a Less version – for this, [Bootstrap v2.3.2's](https://github.com/twbs/bootstrap/releases/tag/v2.3.2) Less mixins and variables are imported in `lib/build.less`; and analog to the Sass mixins, these mixins also include a [`box-shadow` mixin](https://github.com/twbs/bootstrap/blob/b4c4072679943773a7582daa63f545ba818fa627/less/mixins.less#L252).

In conclusion, all is good in regards to these mixins.
As stated above, I'll make sure that the build tasks `master` branch correctly works again – I had to update some Node packages to get Sass to compile – and also update the readme to point users to the bootstrap3 branch.
